### PR TITLE
[ARM Plugin] Ubuntu 20 support

### PR DESCRIPTION
**Pull request by alvoron**
This docker image is the same as for 18 ubuntu except for sources.list content.
I tested it on my 20.04, it works fine, only python 3.7 needs to be installed manually (20.04 comes with 3.8).

### CODE REVIEW

**Review by allnes — APPROVED** [review]

File: modules/arm_plugin/Dockerfile.RPi64_focal [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        --host=aarch64-linux-gnu \
+        --build=x86_64-linux-gnu \
+        --disable-ipv6 \
+        --enable-shared \
+        --prefix=/opt/python3.7_arm \
+        ac_cv_file__dev_ptmx=no \
+        ac_cv_file__dev_ptc=no && \
+    make -j4 && make altinstall
+
+RUN wget https://www.cmake.org/files/v3.13/cmake-3.13.4.tar.gz && \
```
Comment (terfendail) [inline modules/arm_plugin/Dockerfile.RPi64_focal:82]: Ubuntu Focal has cmake 3.16.3 so building from source is unnecessary

**Comment by terfendail** [comment]
Also I think we could try to stick to python 3.8 instead of 3.7 building. However it's not urgent

File: modules/arm_plugin/Dockerfile.RPi64_focal (hunk 1 above)
Comment (alvoron) [reply modules/arm_plugin/Dockerfile.RPi64_focal]: removed cmake compilation

**Comment by alvoron** [comment]
> Also I think we could try to stick to python 3.8 instead of 3.7 building. However it's not urgent

I'll fix it in the next PR. IMO we shouldn't hardcode python-related paths, so the script may pick up any python version correctly.

**Pull request merged**